Gemfile

```diff
@@ -56,11 +56,12 @@ group :development do
   gem 'spring'
 end
 
-group :test do
+group :development, :test do
```

Comment:
- [ ] 37 out of 54 tests are failing when I checked them
![Screenshot from 2021-11-17 22-37-09](https://user-images.githubusercontent.com/62705947/142286442-6fb4a852-2909-438c-9242-c7b205fa4d83.png)